packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/decode/DecodeForm.tsx

```diff
@@ -1,8 +1,9 @@
 "use client";
 
-import { AbiEvent, AbiFunction, toFunctionSelector } from "viem";
+import { AbiFunction, AbiItem, toFunctionSelector } from "viem";
 import { formatAbiItem } from "viem/utils";
 import * as z from "zod";
+import { AbiError } from "abitype";
```

Comment:
does viem not export this?

Reply:
looks like no weirdly enough, cannot import it unlike other types.